### ISSUE TITLE
Get rid of Comparator implementations for SciJava priorities

### ISSUE THREAD

**Issue by imagejan** [mastodon-sc/mastodon-tracking#27]
`PluginProvider` implements its own `Comparator` for SciJava priorities:

https://github.com/mastodon-sc/mastodon-tracking/blob/5b213f9c1b3ad2638929a15d128e645972c0815e/src/main/java/org/mastodon/trackmate/PluginProvider.java#L42-L50

In contrast to the implementation in TrackMate (see https://github.com/fiji/TrackMate/issues/150), the actual comparison is carried out with `priorityComparator.reversed()`, however this isn't necessary since `pluginService.getPluginsOfType( cl )` already returns the plugins sorted by priority, as noted by @ctrueden in https://github.com/fiji/TrackMate/issues/150#issuecomment-583015007.

---

*TL;DR:* let's get rid of the lines 42-50 quoted here.
